setup.py

```diff
@@ -121,7 +121,7 @@
 class NoopTestCommand(TestCommand):
     def __init__(self, dist):
         print("Matplotlib does not support running tests with "
-              "'python setup.py test'. Please run 'python tests.py'.")
+              "'python setup.py test'. Please run 'pytest'.")
```

Comment:
According to [the docs](https://matplotlib.org/devel/testing.html#running-the-tests) you can run the tests with `py.test` or `python tests.py`.  
Should the latter not be the case, one would need to update the docs as well. Else, the error could probably read 

> Please run 'python tests.py' or 'py.test'.

Reply:
I believe there was an attempt to migrate to `py.test`, and to deprecate disincentive `python tests.py`, in particular the documentation does not mention how to run a single test using `python tests.py`. So it seem to me that directly giving the right invocation.

If you look at previous PRs and comment (like https://github.com/matplotlib/matplotlib/pull/8026/files#r101600628), there seem to be some consensus that `pytest` is prefered and should replace `python tests.py` where possible.

Also

```
In [1]: import this
...
There should be one-- and preferably only one --obvious way to do it.
...
```

Reply:
@Carreau all of the pytest CLI args pass through.

Reply:
> ll of the pytest CLI args pass through.

Oh, ok, well the old doc was not saying that, and giving innacurate information. 
Thanks for merging. I think that having pytest everywhere but still have the `python tests.py` as a alias is great !